src/api/googlesheetApi.js

```diff
@@ -1,6 +1,6 @@
 import Tabletop from 'tabletop';
 
-const revere_key = '1QolGVE4wVWSKdiWeMaprQGVI6MsjuLZXM5XQ6mTtONA';
+const revere_key = '1pzTWo43LtmcMl5KSGPtCWCmPfQ9U7-Pp6GnbRBbMWl8';
```

Comment:
Its okay to change the key while developing locally & testing, but in master, we want to preserve the original key.

Reply:
I'll change it back to the original key